RACK-Ontology/ontology/REVIEW.sadl

```diff
@@ -10,15 +10,24 @@
  * of the Defense Advanced Research Projects Agency (DARPA).
  */
 
+/************** edit history *****************
+  *
+  * dwa - 11 Nov 2020 - reduced cardinality of several
+  * dwa - 11 Nov 2020 - removed incorrect associations
+  * dwa - 11 Nov 2020 - removed unnecessary subclassing
+  *
+  *********************************************/
+
 uri "http://arcos.rack/REVIEW" alias Rv.
 import "http://arcos.rack/PROV-S".
+import "http://arcos.rack/DOCUMENT".
 
 REVIEW_LOG
 	(note "Outcome of a REVIEW ACTIVITY")
-	is a type of ENTITY.
+	is a type of REPORT.
```

Comment:
I don't think a  REVIEW_LOG should be a subclass of REPORT, a REPORT would be a collection of  REVIEW_LOGs. For example a REVIEW_LOG would be generated for each REQUIREMENT, then the REPORT would aggregate the REVIEW_LOG items for all the REQUIREMENTs associated with a particular SYSTEM.  This would also allow us to remove the DOCUMENT import